pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1131 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+// This file contains a collection of methods that can be used from go-resful to
+// generate Swagger API documentation for its models. Please read this PR for more
+// information on the implementation: https://github.com/emicklei/go-restful/pull/215
+//
+// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
+// are on one line! For multiple line or blocks that you want to ignore use ---.
+// Any context after a --- is ignored.
+//
+// Those methods can be generated by using hack/update-generated-swagger-docs.sh
+
+// AUTO-GENERATED FUNCTIONS START HERE
+func (AWSElasticBlockStoreVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "AWSElasticBlockStoreVolumeSource represents a Persistent Disk resource in AWS.\n\nAn AWS PD must exist and be formatted before mounting to a container. The disk must also be in the same AWS zone as the kubelet. A AWS PD can only be mounted on a single machine.",
+		"volumeID":  "Unique id of the PD resource. Used to identify the disk in AWS. See http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
```

Comment:
@caesarxuchao im still in the process of reviewing this file but want to change all instances of "See" to "More info". Im nitpicking here because I think prefixing with "More info" better communicates that its optional to follow the link vs. "see" which is authoritative and almost implies that you should follow the link.